deploy/example_clusters/cr-cluster-spark245.yaml

```diff
@@ -5,15 +5,22 @@
 #  labels:
 #    kubedirector.hpe.com/cmType : jupyterhub_ldap_config
 #data:
+## auth_type options are LDAP/Active Directory
+#  jupyterhub_auth_type: Active Directory
+#  jupyterhub_ssl_enabled: "False"
+## use ',' as a delimiter to provide multiple server_address
 #  jupyterhub_server_address: "16.143.20.82"
+#  jupyterhub_ssl_enabled: "False"
 #  jupyterhub_server_port: "389"
 #  jupyterhub_lookup_dn: "True"
 #  jupyterhub_user_search_base: "dc=mip,dc=storage,dc=hpecorp,dc=net"
 #  jupyterhub_user_attribute: "cn"
 #  jupyterhub_bind_user_dn: "cn=admin,dc=mip,dc=storage,dc=hpecorp,dc=net"
 ##  #optional parameter. "use ":" as a delimiter to provide multiple templates"
 #  jupyterhub_bind_dn_template: "cn={username},ou=People,dc=mip,dc=storage,dc=hpecorp,dc=net"
-#  jupyterhub_valid_username_regex: "^[a-z][.@a-z0-9_-]*$"
+#  jupyterhub_valid_username_regex: "^[A-Za-z][.@A-Za-z0-9_-]*$"
+##  "use ":" as a delimiter to provide multiple jupyterhub_member_of"
+#  jupyterhub_member_of: CN=Eng,OU=Engineering,DC=mip,DC=storage,DC=hpecorp,DC=net:CN=QA,OU=Engineering,DC=mip,DC=storage,DC=hpecorp,DC=net
```

Comment:
we should find some other way of getting this info. this is not user-friendly.

Reply:
are we using connection for this?

Reply:
@riteshja  
As discussed, ConfigMap accepts only String. arry gives error.

@kmathur2 
Once configcli with connection feature is available as part of Kubedirector EPIC, code will be updated to use configcli with connections.
need to update kubedirector to use new version of configcli and EPIC to have updated KD (which is having new configcli)